source/fleetprovisioning/FleetProvisioning.cpp

```diff
@@ -460,6 +472,14 @@ bool FleetProvisioning::RegisterThing(Iotidentity::IotIdentityClient identityCli
         return false;
     }
 
+    LOG_INFO(TAG, "Collect system information");
```

Comment:
@HarshGandhi-AWS Is this something we want to enable by default? I was thinking of adding some flag or config setting as a toggle for collecting this information.

Reply:
Nice catch @RogerZhongAWS. @JarryShaw  Enabling collection of system information by default is not recommended since it might contain device sensitive information. I would suggest having an config setting option to enable and disable system information collection which by default should be disabled.

Reply:
Make sense. I'm adding a `collectSystemInformation` flag to it.